modules/oxide/src/less/theme/content/uploadcare/uploadcare.less

```diff
@@ -0,0 +1,29 @@
+@placeholder-initial-width: 600px;
```

Comment:
I wonder if these should be `uploadcare-` prefixed they become quite long though but in case we add some other placeholder this name would then clash. Or do we split then up once we do that into a generic and a specific version of the placeholder?

Reply:
For now these are uploadcare specific and it does make sense to have a prefix so I've decided to add it. If we decide to add more general placeholder with the same styles we can change that